wiki/People/BAT/en.md

```diff
@@ -0,0 +1,84 @@
+# Beatmap Approval Team
+
+The Beatmap Approval Team (_BAT_) are a group of people who are in charge of ranking beatmaps that meet the standards of the [Ranking Criteria](https://github.com/osuTitanic/wiki/blob/main/wiki/Ranking_Criteria/en.md). <!-- change link when official, just dont know how it'll be formatted -->
+<!-- this feels very empty however after sitting on it for a few days I'm not sure what else to add -Nikku -->
+
+
+## Becoming a BAT
+
+Becoming a member of the Beatmap Approval Team requires both a decent amount of knowledge in mapping and understanding of the Ranking Criteria. Potential members must apply while applications are open and are discussed internally for the role. If the majority of the active team agrees that a member has potential, or don't have any issues with their ability, they are added to the team.
+
+
+## Roles and Responsibilities
+
+Members of the Beatmap Approval Team are responsible primarily for managing the moderation of pending beatmaps into the ranking cycle. The BAT are the people to go to for any modding or advice requests about anything mapping or gameplay related.
+
+
+## Current Members
+
+*Please note: All BAT speak English unless otherwise noted.*
+
+
+### Standard Team Members
+
+User | Specialization | Additional Languages Spoken | Added Date 
+---|:---:|:---:|---
+[Adachi](https://osu.titanic.sh/u/39)           |               | Italian                                     | Since the beginning
+[BlueChinchompa](https://osu.titanic.sh/u/40)   |               |                                             | Since the beginning
+[wrn81056](https://osu.titanic.sh/u/645)        |               | French, Malaysian, Russian, German, Chinese | May 19th, 2024[^1] 
+[Digitalfear117](https://osu.titanic.sh/u/809)  | .osu mapping  |                                             | May 29th, 2024
+[Kurboh](https://osu.titanic.sh/u/810)          |               | Croatian                                    | May 22nd, 2024[^3]
+[Nikku](https://osu.titanic.sh/u/811)           | Storyboarding |                                             | May 19th, 2024[^1]
+[-Ady](https://osu.titanic.sh/u/821)            |               |                                             | May 19th, 2024[^1]
+[Oposh](https://osu.titanic.sh/u/829)           |               | Polish                                      | May 19th, 2024[^1]
+[Cartman](https://osu.titanic.sh/u/857)         |               | Tagalog                                     | May 20th, 2024[^2]
+[iSlodinx](https://osu.titanic.sh/u/869)        |               | Serbian, Croatian, Bosnian, some Macedonian | May 29th, 2024
+[arutama](https://osu.titanic.sh/u/905)         |               | Indonesian                                  | June 14th, 2024[^4] 
+[Ignacio](https://osu.titanic.sh/u/943)         |               | Spanish                                     | July 15th, 2024[^5]
```

Comment:
About the "Specialization" tab, should we ask the BAT members for their preferred genres, just like the original wiki post did?

Reply:
I don't remember seeing a preferred genre section from the few archives I looked at, only what types of mapping they were good at, one even labeled "Taiko". maybe I was looking at the wrong ones for that, it could be added but should it replace "Specialization" or have them both?

Reply:
Then we can just leave it like that.
I would have one question - What does `.osu mapping` mean?

Reply:
using a notepad to map instead of the editor; that doesn't need to stay if you think it shouldn't be there

Reply:
We can leave it, I just want to see if there is a more "understandable phrasing", if that makes sense. Perhaps something like "Notepad Mapping", "Beatmap File Expert", etc.